### PR TITLE
Fixes #151. Testing for CASSANDRA-8640

### LINKED ISSUE #151: Expand Paxos Testing
See CASSANDRA-8640

### PR DESCRIPTION
**Pull request by ptnapoleon**
@knifewine for review.

### CODE REVIEW

File: paxos_tests.py [hunk 1]
```diff
@@ -30,6 +31,36 @@ def prepare(self, ordered=False, create_keyspace=True, use_cache=False, nodes=1,
...
+        session = self.prepare(nodes=3, rf=3)
+        session.execute("CREATE TABLE test (k int PRIMARY KEY, v int)")
+        session.execute("INSERT INTO test (k, v) VALUES (0, 0) IF NOT EXISTS")
+
+        self.cluster.nodelist()[2].stop()
+        session.execute("INSERT INTO test (k, v) VALUES (1, 1) IF NOT EXISTS")
+
+        self.cluster.nodelist()[1].stop()
+        assert_unavailable(session.execute, "INSERT INTO test (k, v) VALUES (2, 2) IF NOT EXISTS")
+
```
Comment (knifewine) [inline paxos_tests.py]: This is a maybe a bit peripheral, but might be worthwhile to bring a node back up and make sure we recover the ability to do the insert.

File: paxos_tests.py [hunk 2]
```diff
@@ -30,6 +31,36 @@ def prepare(self, ordered=False, create_keyspace=True, use_cache=False, nodes=1,
...
+        session = self.prepare(nodes=3)
+        session.execute("CREATE TABLE test (k int PRIMARY KEY, v int)")
+        session.execute("INSERT INTO test (k, v) VALUES (0, 0) IF NOT EXISTS")
+
+        self.cluster.nodelist()[2].stop()
+        session.execute("INSERT INTO test (k, v) VALUES (1, 1) IF NOT EXISTS")
+
+        self.cluster.nodelist()[1].stop()
+        session.execute("INSERT INTO test (k, v) VALUES (3, 2) IF NOT EXISTS")
+
```
Comment (knifewine) [inline paxos_tests.py]: Similary here, might be worthwhile to bring nodes back up and retry the operations to see them succeed.

**Comment by ptnapoleon** [comment]
Good suggestions. I will add that today.

**Comment by ptnapoleon** [comment]
@knifewine I have made the suggested modifications, rebased and pushed.

**Comment by knifewine** [comment]
+1 LGTM

**Pull request merged**